source/AI.cpp

```diff
@@ -1829,6 +1829,10 @@ void AI::CircleAround(Ship &ship, Command &command, const Ship &target)
 	command.SetTurn(TurnToward(ship, direction));
 	if(ship.Facing().Unit().Dot(direction) >= 0. && direction.Length() > 200.)
 		command |= Command::FORWARD;
+
+	// Use an equipped afterburner if possible
+	if (command.Has(Command::FORWARD) && direction.Length() > 600 && ShouldUseAfterburner(ship))
```

Comment:
Are there cases where the ship will have `FORWARD` set but not from the block immediately above? If not, we should probably bundle this check inside that conditional block.

I also think that a more stringent facing alignment with the desired direction of travel could be warranted.

Reply:
Ping @lukearndt

Reply:
My passion for the project ran dry a while ago. You're very welcome to push additional commits to the branch if you'd like to get it to a state where you're happy to merge it. 😄